docs/source/user-guide/models.rst

```diff
@@ -1,2 +1,178 @@
+:doc:`Metrics <metrics>` for more information on the metrics
+on which the model is tested in ``Trainer.test()``.
+
+Below is an example of fitting and testing a model with a given data module.
+
+.. code-block:: python
+    :linenos:
+
+    from climate_learn.training import Trainer, WandbLogger
+
+    trainer = Trainer(
+        seed = 0,
+        accelerator = "gpu",
+        precision = 16,
+        max_epochs = 5,
+        logger = WandbLogger(project = "climate-learn-test", name = "forecast-vit"),
+    )
+
+    trainer.fit(model_module, data_module)
+
+    trainer.test(model_module, data_module)
+
+Example
+-------
+
+The following can be run in Google Colab.
+
+.. nbinput:: ipython3
+    :execution-count: 1
+
+    %%capture
+    !pip install git+https://github.com/aditya-grover/climate-learn.git
+
+.. nbinput:: ipython3
+    :execution-count: 2
+
+    # Download WeatherBench 2m_temperature data to Google Drive
+    from google.colab import drive
+    from climate_learn.data import download
+
+    drive.mount("/content/drive")    
+    download(
+        root="/content/drive/MyDrive/Climate/.climate_tutorial",
+        source="weatherbench",
+        variable="2m_temperature",
+        dataset="era5", 
+        resolution="5.625"
+    )
+
+.. nbinput:: ipython3
+    :execution-count: 3
+
+    # Load data module for forecasting task
+    from climate_learn.utils.datetime import Year, Days, Hours
+    from climate_learn.data import DataModule
+
+    data_module = DataModule(
+        dataset = "ERA5",
+        task = "forecasting",
+        root_dir = "/content/drive/MyDrive/Climate/.climate_tutorial/data/weatherbench/era5/5.625/",
+        in_vars = ["2m_temperature"],
+        out_vars = ["2m_temperature"],
+        train_start_year = Year(1979),
+        val_start_year = Year(2015),
+        test_start_year = Year(2017),
+        end_year = Year(2018),
+        pred_range = Days(3),
+        subsample = Hours(6),
+        batch_size = 128,
+        num_workers = 1
+    )
+
+.. nbinput:: ipython3
+    :execution-count: 4
+
+    # Load U-Net model
+    from climate_learn.models import load_model
+
+    model_kwargs = {
+        "in_channels": len(data_module.hparams.in_vars),
+        "out_channels": len(data_module.hparams.out_vars),
+        "n_blocks": 4
+    }
+
+    optim_kwargs = {
+        "lr": 1e-4,
+        "weight_decay": 1e-5,
+        "warmup_epochs": 1,
+        "max_epochs": 5,
+    }
+
+    model_module = load_model(
+        name="unet",
+        task="forecasting",
+        model_kwargs=model_kwargs,
+        optim_kwargs=optim_kwargs
+    )
+
+.. nbinput:: ipython3
+    :execution-count: 5
+
+    from climate_learn.training import Trainer, WandbLogger
+
+    # Initialize model trainer
+    trainer = Trainer(
+        seed = 0,
+        accelerator = "gpu",
+        precision = 16,
+        max_epochs = 5,
+        # logger = WandbLogger(project = "climate-learn-test", name = "forecast-unet"),
```

Comment:
Remove wandb. Also generally, idt we should leave commented-out lines of code in the docs.

Reply:
removed